system/gatekeeper/lib/helm-release.rego

```diff
@@ -43,6 +44,23 @@ __is_helm_release(obj) = false {
   obj.type != "helm.sh/release.v1"
 }
 
+
+__should_be_checked(obj) = true {
+  obj.metadata.labels.status == "deployed"
+}
+__should_be_checked(obj) = true {
+  obj.metadata.labels.status == "pending-upgrade"
+}
+__should_be_checked(obj) = false {
+  obj.metadata.labels.status == "superseded"
+}
+__should_be_checked(obj) = false {
+  obj.metadata.labels.status != "deployed"
+  obj.metadata.labels.status != "pending-upgrade"
+  obj.metadata.labels.status != "superseded"
+}
+
+
```

Comment:
Is this necessary? This should already be enforced by the match selector on the constraints.